client/core/trade_simnet_test.go

```diff
@@ -1293,34 +1348,207 @@ func tryUntil(ctx context.Context, tryDuration time.Duration, tryFn func() bool)
 HELPER TYPES, FUNCTIONS AND METHODS
 ************************************/
 
+// marketConfig taken from dcrdex/server/cmd/dcrdex/settings.go
+type marketConfig struct {
+	Markets []*struct {
+		Base           string  `json:"base"`
+		Quote          string  `json:"quote"`
+		LotSize        uint64  `json:"lotSize"`
+		RateStep       uint64  `json:"rateStep"`
+		Duration       uint64  `json:"epochDuration"`
+		MBBuffer       float64 `json:"marketBuyBuffer"`
+		BookedLotLimit uint32  `json:"userBookedLotLimit"`
+	} `json:"markets"`
+	Assets map[string]*dexsrv.AssetConf `json:"assets"`
+}
+
+type harnessCtrl struct {
+	tmuxWindow, fundStr string
+	blockMultiplier     uint32
+}
+
+func (hc *harnessCtrl) tmuxRun(cmd string) error {
+	cmd += "; tmux wait-for -S harnessdone"
+	err := exec.Command("tmux", "send-keys", "-t", hc.tmuxWindow, cmd, "C-m").Run() // ; wait-for harnessdone
+	if err != nil {
+		return nil
+	}
+	return exec.Command("tmux", "wait-for", "harnessdone").Run()
+}
```

Comment:
Instead of using tmux, with the harnessdone signal, it's easier to just call the command directly from the appropriate dextest directory. 

See e.g. `func harnessCtl` in **loadbot.go**.